tests/functional/test_clidriver.py

```diff
@@ -27,6 +27,17 @@
 from awscli.clidriver import create_clidriver
 
 
+class RegionCapture(object):
+    def __init__(self):
+        self._region = None
```

Comment:
Make this public since we are accessing it from outside the class (in the test).